gocd/deploy.pipeline.gocd.yml

```diff
@@ -8,11 +8,6 @@ common:
           source: terraform/certs_deployment.tfplan
           destination: terraform/
     tasks:
-      - exec:
-          command: /bin/bash
-          arguments:
-            - -c
-            - ./tasks create_secrets
       - exec:
```

Comment:
Is this `./tasks tf_plan_certs create` step needed now?

Reply:
Yes, because it doesn't do certs, it creates DynamoDB IAM permissions!